x-pack/legacy/plugins/apm/public/components/app/ServiceMap/Cytoscape.tsx

```diff
@@ -52,30 +56,65 @@ function useCytoscape(options: cytoscape.CytoscapeOptions) {
   return [ref, cy] as [React.MutableRefObject<any>, cytoscape.Core | undefined];
 }
 
+function getLayoutOptions(
+  selectedRoots: string[],
+  height: number,
+  width: number
+): cytoscape.LayoutOptions {
+  return {
+    name: 'breadthfirst',
+    roots: selectedRoots,
+    fit: true,
+    padding: nodeHeight,
+    nodeDimensionsIncludeLabels: true,
+    animate: true,
+    animationEasing: animationOptions.easing,
+    animationDuration: animationOptions.duration,
+    // Rotate nodes from top -> bottom to display left -> right
+    // @ts-ignore
```

Comment:
You can put a description of why we're ignoring something after the `@ts-ignore` on the same line comment.

Reply:
@smith great idea. 
What about opening a PR to fix [the types](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/cytoscape/index.d.ts#L4621-L4627) and link to that in the comment?

Reply:
@sqren I'll just wait until this is merged and open a PR that updates the types and removes the comments that can be removed.